bokehjs/src/coffee/common/embed.coffee

```diff
@@ -8,6 +8,18 @@ HasProperties = require "./has_properties"
 {pull_session} = require "./client"
 {Promise} = require "es6-promise"
 
+_handle_notebook_comms = (msg) ->
+  logger.debug("handling notebook comms")
+  # @ is bound to the doc
+  @apply_json_patch_string(msg.content.data)
+
+_init_comms = (target, doc) ->
+  logger.info("Initializing Jupyter comms for target #{target} and doc #{doc}")
+  comm_manager = Jupyter.notebook.kernel.comm_manager
```

Comment:
I will try to research on this later... but I certainly do not expect the removal of Jupyter namespace anytime soon... even more if there are cases that actually need the namespace.